pandas/_libs/tslibs/conversion.pyx

```diff
@@ -1015,7 +1031,7 @@ def tz_localize_to_utc(ndarray[int64_t] vals, object tz, object ambiguous=None,
                 stamp = _render_tstamp(val)
                 raise pytz.NonExistentTimeError(stamp)
 
-    return result
+    return result.base  # `.base` to access underlying np.array
```

Comment:
same

Reply:
See comment above.  I previously changed all of the `foo.base` to `np.asarray(foo)`, but reverted that change after finding that risked incorrectly returning `np.array(None)` instead of raising if `None` gets passed to one of these functions.

Reply:
from the docs

https://cython.readthedocs.io/en/latest/src/userguide/memoryviews.html

```
import numpy as np

def process_buffer(int[:,:] input_view not None,
                   int[:,:] output_view=None):

   if output_view is None:
       # Creating a default view, e.g.
       output_view = np.empty_like(input_view)

   # process 'input_view' into 'output_view'
   return output_view
```
you can use ``not None`` in the declaration to avoid this issue.

Reply:
Yah, I was hoping you had forgotten about that.  Part of the goal ATM is to move the cython code closer to being valid python (linting!), and `not None` moves that in the wrong direction.  Will un-revert, and open an issue with Cython on implementing a py-friendly version of `not None`.  Ditto for the other open cython PR.

Reply:
well I actually find it weird that cython code accepts None here at all by default.